python/cugraph-dgl/setup.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
```

Comment:
I've made some important changes to this file for building the conda package [here](https://github.com/rapidsai/cugraph/blob/8bfcbd5f21742049056bf5fc298d613db170568b/python/cugraph-dgl/setup.py). I'm open to suggestion on how we should coordinate this wrt who merges first.

Reply:
Lets merge this PR in as it includes the most changes and has the things we all need.